config_parser/config_reader.py

```diff
@@ -0,0 +1,15 @@
+"""
+Module for getting value of certain parameter in certain section in cfg. file
+"""
+import ConfigParser
+
+
+def get_setting(cfg_file, section,  parameter):
+    """
+    cfg_file - name or path and name of cfg file for parsing
```

Comment:
It would be great to follow:
http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html